### ISSUE TITLE
macros: bug or feature?

### ISSUE THREAD

**Issue by ThomasBreuer** [oscar-system/GAP.jl#517]
Is the following behaviour something that should be documented as a feature or a known deficiency of macros, or should the behaviour be changed?
```
julia> using GAP
[...]

julia> GAP.julia_to_gap( "abc\ndef" )   # this is correct
GAP: "abc\ndef"

julia> x = @gap "\"abc\\ndef\""   # this is also correct
GAP: "abc\ndef"

julia> g"abc\ndef"   # this is not what one really expects
GAP: "abc\\ndef"

```
(I do not like these maros at all.)

**Comment by fingolfin** [comment]
I'd consider the behavior of the 3rd example to be a bug.

**Comment by ThomasBreuer** [comment]
@fingolfin I will create a pull request for fixing the 3rd example.

**Comment by ThomasBreuer** [comment]
can be closed because of #525
